lib/EnsEMBL/REST/Controller/VEP.pm

```diff
@@ -353,6 +356,17 @@ sub _include_user_params {
   return \%vep_params;
 }
 
+sub _amend_transcript_id {
+  my ($self, $config) = @_;
+
+  if(exists($config->{transcript_id}) && length($config->{transcript_id}) < 25 && not ($config->{transcript_id} =~ m/[^A-Z0-9_.]/i))
```

Comment:
This is perl... we use ! ;)
Nah, we've discussed this in Core, just for consistency, perhaps change not to ! as you're using && just before rather than and. And while you're at it, please wrap that line, it's longer than the github viewer, again for readability in the long term.
Nitpicky, but a simple change. Thanks.